src/render_tasks/d3d12_deferred_composition.cpp

```diff
@@ -98,10 +105,10 @@ namespace wr
 			data.in_pipeline = (D3D12Pipeline*)ps_registry.Find(pipelines::deferred_composition);
 
 			// Check if the current frame graph contains the hybrid task to know if it is hybrid or not.
-			data.is_hybrid = fg.HasTask<wr::RTHybridData>();
 			data.is_path_tracer = fg.HasTask<wr::PathTracerData>();
 			data.is_rtao = fg.HasTask<wr::RTAOData>();
 			data.is_hbao = fg.HasTask<wr::HBAOData>() && !data.is_rtao; //Don't use HBAO when RTAO is active
+			data.is_hybrid = fg.HasTask<wr::RTShadowData>() || fg.HasTask<wr::RTHybridData>() || fg.HasTask<wr::RTReflectionData>() || fg.HasTask<wr::ShadowDenoiserData>();
```

Comment:
is hybrid doesn't suffice anymore, you probably need a boolean for shadow and reflections now.

Reply:
I already have those booleans, setting them later in the code. is_hybrid is more of an indicator that says if raytracing in general is used. I'll need to actually sent has_reflections and has_shadows to the gpu though, adding that.